en/controllers/components.rst

```diff
@@ -146,7 +146,7 @@ controller, you could access them like so::
 
         public function delete()
         {
-            if ($this->Post->delete($this->request->data('Post.id')) {
+            if ($this->Post->delete($this->request->getData('Post.id')) {
```

Comment:
i don't think versionadded works in a code block. I also don't think that sphinx can output versioned code blocks without a custom extension.